client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -115,6 +115,15 @@ class StatModuleChartTabs extends Component {
 			showChartHeader = false,
 			isNewDateFilteringEnabled = false,
 		} = this.props;
+
+		let chartData = this.props.chartData;
```

Comment:
By making the `queryDate` parameter optional to the `buildChartData` function, we may be able to [ignore](https://github.com/Automattic/wp-calypso/blob/250ddab07aa547da0df430c01cc4a9920094ebfb/client/my-sites/stats/stats-chart-tabs/utility.js#L60) the `is-selected` class.

```
const chartData = buildChartData(
	activeLegend,
	chartTab,
	counts,
	period,
	! isNewDateFilteringEnabled ? queryDate : null
);
```